frontend/www/js/omegaup/graderv2/ephemeral.ts

```diff
@@ -314,48 +113,18 @@ RegisterVueComponent(
 
 function initialize() {
   layout.init();
-
   if (isEmbedded) {
     // Embedded layout should not be able to modify the settings.
-    layout.root.getItemsById('settings')[0].remove();
     document.getElementById('download').style.display = 'none';
     document.getElementById('upload').style.display = 'none';
     document.querySelector('label[for="upload"]').style.display = 'none';
 
-    // Since the embedded grader has a lot less horizontal space available, we
-    // move the first two columns into a stack so they can be switched between.
-    let mainColumn = layout.root.getItemsById('main-column')[0];
-    let casesColumn = layout.root.getItemsById('cases-column')[0];
-    let caseSelectorColumn = layout.root.getItemsById(
-      'case-selector-column',
-    )[0];
-    let oldWidth = caseSelectorColumn.element[0].clientWidth;
-    let oldHeight = caseSelectorColumn.element[0].clientHeight;
-
-    let newStack = layout.createContentItem({
-      type: 'stack',
-      content: [],
-      isClosable: false,
-    });
-
-    mainColumn.parent.addChild(newStack, 0);
-
-    casesColumn.setTitle('cases');
-    casesColumn.parent.removeChild(casesColumn, true);
-    newStack.addChild(casesColumn, 0);
-
-    mainColumn.setTitle('code');
-    mainColumn.parent.removeChild(mainColumn, true);
-    newStack.addChild(mainColumn, 0);
-
-    // Also extend the case selector column a little bit so that it looks nicer.
-    caseSelectorColumn.container.setSize(Math.max(160, oldWidth), oldHeight);
-
     // Whenever a case is selected, show the cases tab.
     store.watch(
       Object.getOwnPropertyDescriptor(store.getters, 'currentCase').get,
       (value) => {
         if (store.getters.isUpdatingSettings) return;
+        let casesColumn = layout.root.getItemsById('cases-column')[0];
```

Comment:
Thank you for moving this line closer to where it will be used. Just a small comment: when a value is not going to change, we can use `const` instead of `let`.
```suggestion
        const casesColumn = layout.root.getItemsById('cases-column')[0];
```